src/main/java/org/broadinstitute/hellbender/utils/pileup/PileupElement.java

```diff
@@ -90,6 +92,31 @@ public static PileupElement createPileupForReadAndOffset(final GATKRead read, fi
                 " but we never saw such an offset in the alignment state machine");
     }
 
+    /**
+     * Create a pileup element for read at offset.
+     *
+     * offset must correspond to a valid read offset given the read's cigar, or an IllegalStateException will be throw
+     *
+     * @param read a read
+     * @param loc A one base location you want to generate your genome
+     * @return a valid PileupElement with read and at offset
+     */
+    public static PileupElement createPileupForReadAndOffset(final GATKRead read, final Locatable loc) {
```

Comment:
it seems like this should just delagate to the existing one.

Reply:
Ive renamed the method to help reduce confusion